### PR TITLE
Add new LXC: Scrutiny

### PR DESCRIPTION
**Pull request by MickLesk**
## Description
WebUI for smartd S.M.A.R.T monitoring

That's a big script.
Here I have differentiated between WebApp and the Collector (without UI).
I have tested the script at all points, both the update script and the starting of the individual services.

As the script is quite large, I assume that you can think of a few more adjustments. I wanted to make it as "all in one" as possible. 

image
image

Please delete options that are not relevant.
- [x] New Script (Develop a new script or set of scripts that are fully functional and thoroughly tested)
- [x] I have performed a self-review of my code, adhering to established codebase patterns and conventions.
- [x] This change requires a documentation update

### CODE REVIEW

**Comment by tteck** [comment]
Why not just install the GUI version, and put the other stuff in the "Scrutiny Management" section?

**Comment by MickLesk** [comment]
> Why not just install the GUI version, and put the other stuff in the "Scrutiny Management" section?

 yeah, thats also possible. Then I switch a bit and reduce the number of lines in the main script

**Comment by tteck** [comment]
"Setup InfluxDB-Connection" should be done manually in the `scrutiny.yaml` instead of prompts.

**Comment by MickLesk** [comment]
> "Setup InfluxDB-Connection" should be done manually in the `scrutiny.yaml` instead of prompts.

Do you mean that I delete the whole prompt area and the users (of your repo) should do this manually via nano? Or am I misunderstanding this?

**Comment by tteck** [comment]
You're correct

**Comment by MickLesk** [comment]
Im still making a few tweaks, I'll comment as shortly as I've got a final version

**Comment by MickLesk** [comment]
you can check it. ive added an 4. whiptail option to direct edit the config-file

File: ct/scrutiny.sh [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+  SSH="no"
+  VERB="no"
+  echo_default
+}
+
+function update_script() {
+  if [[ ! -d /opt/scrutiny ]]; then
+    msg_error "No ${APP} Installation Found!"
+    exit
+  fi
```
Comment (tteck) [inline ct/scrutiny.sh]: Single line
`if [[ ! -d /opt/scrutiny ]]; then msg_error "No ${APP} Installation Found!"; exit; fi`

File: ct/scrutiny.sh [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+  UPD=$(whiptail --backtitle "Proxmox VE Helper Scripts" --title "Scrutiny Management" --radiolist --cancel-button Exit-Script "Spacebar = Select" 15 70 4 \
+    "1" "Update Scrutiny to $RELEASE" ON \
+    "2" "Start Scrutiny Webapp" OFF \
+    "3" "Create/Start Scrutiny Collector" OFF \
+	"4" "Change Scrutiny Settings"  OFF \
+    3>&1 1>&2 2>&3)
+  header_info
+
+  if [ "$UPD" == "1" ]; then
+    if [[ "${RELEASE}" != "$(cat /opt/scrutiny_version.txt)" ]] || [[ ! -f /opt/scrutiny_version.txt ]]; then
```
Comment (tteck) [inline ct/scrutiny.sh]: `if [[ ! -f /opt/${APP}_version.txt ]] || [[ "${RELEASE}" != "$(cat /opt/${APP}_version.txt)" ]]; then`

File: install/scrutiny-install.sh [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  curl \
+  smartmontools  \
+  make \
+  mc
+msg_ok "Installed Dependencies"
+
+msg_info "Installing Scrutiny WebApp"
+mkdir -p /opt/scrutiny/config
+mkdir -p /opt/scrutiny/web
+mkdir -p /opt/scrutiny/bin
```
Comment (tteck) [inline install/scrutiny-install.sh]: `mkdir -p /opt/scrutiny/{config,web,bin}`

File: install/scrutiny-install.sh [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+RELEASE=$(curl -s https://api.github.com/repos/analogj/scrutiny/releases/latest | grep "tag_name" | awk '{print substr($2, 2, length($2)-3) }')
+echo "${RELEASE}" >/opt/scrutiny_version.txt
```
Comment (tteck) [inline install/scrutiny-install.sh]: `echo "${RELEASE}" >/opt/${APPLICATION}_version.txt`

File: install/scrutiny-install.sh [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+wget -q -O /opt/scrutiny/config/scrutiny.yaml https://raw.githubusercontent.com/AnalogJ/scrutiny/master/example.scrutiny.yaml
+wget -q -O /opt/scrutiny/bin/scrutiny-web-linux-amd64 "https://github.com/AnalogJ/scrutiny/releases/download/${RELEASE}/scrutiny-web-linux-amd64"
+wget -q -O /opt/scrutiny/web/scrutiny-web-frontend.tar.gz "https://github.com/AnalogJ/scrutiny/releases/download/${RELEASE}/scrutiny-web-frontend.tar.gz"
+cd /opt/scrutiny/web && tar xzf scrutiny-web-frontend.tar.gz --strip-components 1 -C .
```
Comment (tteck) [inline install/scrutiny-install.sh]: Stay away from `&&` when possible

File: install/scrutiny-install.sh [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+[Service]
+Type=simple
+ExecStart=/opt/scrutiny/bin/scrutiny-web-linux-amd64 start --config /opt/scrutiny/config/scrutiny.yaml
+Restart=always
+User=root
+
+[Install]
+WantedBy=multi-user.target
+EOF
+    systemctl enable -q --now scrutiny.service
```
Comment (tteck) [inline install/scrutiny-install.sh]: double check all indentations

File: install/scrutiny-install.sh (hunk 4 above)
Comment (tteck) [reply install/scrutiny-install.sh]: I just fixed this https://github.com/tteck/Proxmox/issues/3269 you used `${APP}` instead of `${APPLICATION}`

**Pull request merged**